src/libDirectoryService/DirectoryService.h

```diff
@@ -178,6 +180,7 @@ class DirectoryService : public Executable, public Broadcastable
     void ComputeSharding(
         const vector<pair<array<unsigned char, 32>, PubKey>>& sortedPoWSolns);
     void ComputeTxnSharingAssignments(const Peer& winnerpeer);
+    bool VerifyPoWOrdering();
```

Comment:
Should this be called as well by the DS backup?

Reply:
it is being called. DSBlockPreProcessing.cpp

Reply:
Oops missed that